safe/gui/tools/wizard/step_kw30_field.py

```diff
@@ -134,16 +135,25 @@ def on_lstFields_itemSelectionChanged(self):
             # Generate description for the field.
             field_type = layer_fields.field(field_name).typeName()
             field_index = layer_fields.indexFromName(field_name)
-            unique_values = self.parent.layer.uniqueValues(field_index)[0:48]
+            unique_values = self.parent.layer.uniqueValues(field_index)
             unique_values_str = [
                 i is not None and unicode(i) or 'NULL'
-                for i in unique_values]
+                for i in unique_values[0:48]]
             unique_values_str = ', '.join(unique_values_str)
             field_descriptions += tr('<b>Field name</b>: {field_name}').format(
                 field_name=field_name)
             field_descriptions += tr(
                 '<br><b>Field type</b>: {field_type}').format(
                 field_type=field_type)
+            if (feature_count != -1 and (
+                        layer_purpose == layer_purpose_aggregation)):
+                if len(unique_values) == feature_count:
+                    unique = tr('Yes')
+                else:
+                    unique = tr('No')
+                field_descriptions += tr(
+                    '<br><b>Unique</b>: %s (%d unique values from %d '
```

Comment:
Shame, of course. I always support to use new format, but it seems I forget about it.
Thanks